eq-author/src/App/page/Design/QuestionPageEditor/AnswersEditor/index.js

```diff
@@ -18,7 +19,7 @@ export const AnswersEditor = ({
   onDeleteAnswer,
   moveAnswer,
 }) => {
-  return (
+  return isEmpty(answers.length) ? (
```

Comment:
Should this be `isEmpty(answers)`?

Reply:
Actually, should the logic be reversed? `!isEmpty(answers)`

Reply:
now reverted. don't think its needed anymore.